core/libp2p/crypto/common.hpp

```diff
@@ -19,16 +19,16 @@ namespace libp2p::crypto::common {
    * Values for AES-128
    */
   struct Aes128Secret {
-    uint8_t key[8];
-    uint8_t iv[8];
+    uint8_t key[16];
```

Comment:
Maybe, these numbers can be extracted to named constants, so that client creating `Aes128Secret` could write something like `Aes128Secret{[Aes128Secret::KeySize]{0x1, 0x2...}, ...`? The same applies to 256 secret

Reply:
Maybe defining such constants is a good idea, but I doubt can I say the same about specified usage.

Reply:
Look what I've done.